src/components/LocationSelector/LocationSelector.js

```diff
@@ -83,13 +81,13 @@ function LocationSelectorWithGoogleLoader(props) {
         const { Geocoder } = window.google.maps;
         const geocoder = new Geocoder();
 
-        findCenter(geocoder, location.place_id)
+        return findCenter(geocoder, location.place_id)
             .then(center => {
                 const locationToAdd = {
                     ...location,
                     center: {
-                        lng: center.lng(),
-                        lat: center.lat(),
+                        lng: typeof center.lng === 'function' ? center.lng() : center.lng,
+                        lat: typeof center.lat === 'function' ? center.lat() : center.lat,
```

Comment:
I forgot, what's the point of having `lat ` `lng` as functions?

Reply:
Sometimes google returns it as a component/function. I'm also a bit confused when is which, but this way we are sure to capture it.